netmaster/objApi/apiController.go

```diff
@@ -382,6 +378,10 @@ func (ac *APIController) EndpointGroupCreate(endpointGroup *contivModel.Endpoint
 		return err
 	}
 
+	// Setup links
+	modeldb.AddLink(&endpointGroup.Links.Tenant, tenant)
+	modeldb.AddLinkSet(&tenant.LinkSets.EndpointGroups, endpointGroup)
+
```

Comment:
Two comments:
1. Please move tenant.Write() after "modeldb.AddLinkSet(&tenant.LinkSets.EndpointGroups, endpointGroup)" since it modifies the tenant's linkset.
2. This change still leaves stale link sets if master.PolicyAttach() fails. Can we handle that too